DOCS/man/vo.rst

```diff
@@ -363,6 +363,76 @@ Available video output drivers are:
     ``--vo-tct-256=<yes|no>`` (default: no)
         Use 256 colors - for terminals which don't support true color.
 
+``sixel``
+    Sixel graphics video output driver based on libsixel that works on a
+    console that has sixel graphics enabled such as ``xterm`` or ``mlterm``.
+    Additionally some terminals have limitation on the dimensions, so may
+    not display images bigger than 1000x1000 for example. Make sure that
+    ``img2sixel`` can display images of the corresponding resolution.
+
+    Since mpv 0.30.0, you may need to use ``--profile=sw-fast`` to get decent
+    performance.
+
+    Note: the Sixel image output is not synchronized with other terminal output
+    from mpv, which can lead to broken images. The option ``--really-quiet``
+    can help with that, and is recommended.
+
+    ``--vo-sixel-diffusion=<algo>``
+        Selects the diffusion algorithm for dithering used by libsixel.
+        Can be one of the below list as per libsixel's documentation.
+
+        auto
+            Choose diffuse type automatically
+        none
+            Don't diffuse
+        atkinson
+            Diffuse with Bill Atkinson's method. (Default)
+        fs
+            Diffuse with Floyd-Steinberg method
+        jajuni
+            Diffuse with Jarvis, Judice & Ninke method
+        stucki
+            Diffuse with Stucki's method
+        burkes
+            Diffuse with Burkes' method
+        arithmetic
+            Positionally stable arithmetic dither
+        xor
+            Positionally stable arithmetic xor based dither
+
+    ``--vo-sixel-width=<width>``  ``--vo-sixel-height=<height>``
+        The output video resolution will be set to width and height
+        These default to 320x240 if not set. The terminal window must
```

Comment:
How can the user ensure that the terminal size in pixel is bigger than this? typically the user can tell the size in character cells, but not in pixels, no?

Reply:
Yes it is difficult to ensure in pixels, but the user can then drag and resize their terminal window to get the optimal window without stuttering. It is just recommended to have bigger window otherwise especially if the height is bigger then there is flickering effects due to sixel.

Reply:
See my earlier comment on the subject, but in general - mainly that I don't know if it's possible or easy enough, but I think that if we can change the dimensions to be char-cell-based rather than pixels, then it would be a more natural fit for a terminal output.